### PR TITLE
martian: fix CONNECTs content length 

### PR DESCRIPTION
**Pull request by Choraden**

### CODE REVIEW

File: internal/martian/flush.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 )
 
 func shouldFlush(res *http.Response) bool {
-	if res.Request.Method == http.MethodHead {
+	if res.Request.Method == http.MethodHead || res.Request.Method == http.MethodConnect {
```
Comment (mmatczuk) [inline internal/martian/flush.go:26]: I'm not sure this should be here.

Failed connect is free to return response as any other method.

**Comment by mmatczuk** [comment]
Good catch please add a test.

File: internal/martian/flush.go (hunk 1 above)
Comment (Choraden) [reply internal/martian/flush.go]: OK, I will drop that.

**Comment by Choraden** [comment]
Tests were added.

**Review by mmatczuk — APPROVED** [review]
LGTM

**Pull request merged**